spec/controllers/concerns/reauthentication_required_concern_spec.rb

```diff
@@ -1,19 +1,26 @@
 require 'rails_helper'
 
-describe ReauthnRequiredController do
+describe ReauthenticationRequiredConcern do
   let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }
 
+  class ReauthenticationRequiredController < ApplicationController
+    include ReauthenticationRequiredConcern
+
+    before_action :confirm_recently_authenticated
+
+    def show
+      render plain: 'Hello'
+    end
+  end
+
   describe '#confirm_recently_authenticated' do
-    controller do
-      def show
-        render plain: 'Hello'
-      end
+    controller ReauthenticationRequiredController do
     end
```

Comment:
Worth inlining the contents of the class above here?

Alternatively, I might expect the standard style we use for empty bodies to look more like...

```suggestion
    controller ReauthenticationRequiredController do; end
```

Reply:
rubocop disagrees 😭

```
spec/controllers/concerns/reauthentication_required_concern_spec.rb:17:51: C: Style/BlockDelimiters: Prefer {...} over do...end for single-line blocks.
    controller ReauthenticationRequiredController do; end
```

Reply:
Hm, I think I was getting it confused with empty methods.

https://github.com/18F/identity-idp/blob/a95f829ba81773628c2db37552dc5db5419042a1/app/controllers/account_reset/pending_controller.rb#L13